.github/workflows/static.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     name: ubuntu-static-tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
why this?

Reply:
@XVilka ^

Reply:
I wrote already - It was the ancient GCC/Binutils that Ubuntu 18.04 LTS used. Modern GCC-9 fixed those LTO problems. Switching to Ubuntu 20.04 helps.